AllReadyApp/Web-App/AllReady.UnitTest/Hangfire/Jobs/SendRequestStatusToGetASmokeAlarmShould.cs

```diff
@@ -26,25 +27,36 @@ public SendRequestStatusToGetASmokeAlarmShould()
         }
 
         [Fact]
-        public void SendJsonService()
+        public void SendsValidHttpRequestMessage()
         {
-            HttpRequestMessage requestMsg = null;
             var mockedHttpClient = new Mock<IHttpClient>();
-            mockedHttpClient.Setup(x => x.SendAsync(It.IsAny<HttpRequestMessage>()))
-                .ReturnsAsync(new HttpResponseMessage())
-                .Callback<HttpRequestMessage>(request => requestMsg = request);
             var sendRequestStatusToGetASmokeAlarm = new SendRequestStatusToGetASmokeAlarm(_getASmokeAlarmApiSettings, mockedHttpClient.Object);
             try
```

Comment:
Because the response object is not mocked, the send will throw an exception on `response.EnsureSuccessStatusCode();` . That's why try-catch is used. We could either mock a response or use try catch.